lua/plugins/configs/lspconfig.lua

```diff
@@ -32,8 +32,8 @@ local function on_attach(_, bufnr)
    buf_set_keymap("n", "<space>ca", "<cmd>lua vim.lsp.buf.code_action()<CR>", opts)
    buf_set_keymap("n", "gr", "<cmd>lua vim.lsp.buf.references()<CR>", opts)
    buf_set_keymap("n", "<space>e", "<cmd>lua vim.lsp.diagnostic.show_line_diagnostics()<CR>", opts)
-   buf_set_keymap("n", "[d", "<cmd>lua vim.lsp.diagnostic.goto_prev()<CR>", opts)
-   buf_set_keymap("n", "]d", "<cmd>lua vim.lsp.diagnostic.goto_next()<CR>", opts)
+   buf_set_keymap("n", "[d", "<cmd>lua vim.lsp.diagnostic.goto_prev()<CR>zz", opts)
```

Comment:
This seems like a good & reasonable idea!! 😄

Reply:
what does the zz mean here?

Reply:
`zz` will try to focus your current line,

IE. it will try to scroll to position your current line in the center of the screen

Reply:
@G-Rowell all of the PR is mostly user specific! and ig it'd be better to have lspconfig's default mappings 

![image](https://user-images.githubusercontent.com/59060246/135055665-e522e6dd-d9c3-4dee-8655-9cce077d12e3.png)


If the user doesnt want it then the overriding default config feature could be used , since #465 has merged ._.